js/HelpTextComponent.js

```diff
@@ -156,4 +167,17 @@ HelpTextComponent.prototype.handleFocusOut = function (e) {
     }
 }
 
+/**
+ * Toggle focusability of elements
+ */
+HelpTextComponent.prototype.toggleChildElementInteractivity = function ($container, interactive) {
+    var focusableElements = 'a[href], button';
```

Comment:
We've got two lists of elements now, might be useful to have this as some form of high-level config var? 

https://github.com/jadu/pulsar/blob/develop/js/TableDetailComponent.js#L10

Reply:
Possibly yeah, I think I've got a similar situation in the next issue I'm going to pick up related to the main nav so can investigate there.